OpenRA.Game/Traits/RevealsShroud.cs

```diff
@@ -15,13 +15,17 @@ namespace OpenRA.Traits
 	public class RevealsShroudInfo : ITraitInfo
 	{
 		public readonly WRange Range = WRange.Zero;
+		[Desc("Minimum delay in ticks before next visibility update")]
+		public readonly int VisibilityUpdateDelay = 1;
```

Comment:
I think `Visibility` is implicit here seeing as this is dealing with revealing parts of the map.